terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl

```diff
@@ -1190,7 +1190,15 @@ prometheus:
     ## The scrape configuraiton example below will find master nodes, provided they have the name .*mst.*, relabel the
     ## port to 2379 and allow etcd scraping provided it is running on all Kubernetes master nodes
     ##
-    additionalScrapeConfigs: []
+    additionalScrapeConfigs:
```

Comment:
Since you've enabled the `ServiceMonitor` in the helm chart, we shouldn't need this extra config block here.

Reply:
A very good point. Reverted changed and padded out service monitor creation in values file. 
3b099f3